### ISSUE TITLE
No logs on setup crash

### ISSUE THREAD

**Issue by sloshy** [coursier/coursier#1963]
Hello, I am trying to use the coursier CLI on a machine that requires some specific proxy or resolver settings, as there are a lot of sites blocked on our internal network. I have figured out fetching artifacts from the company artifactory instance, but whenever I run `cs setup`, it crashes without an error message. As seen here:

```
➜  ~ ./cs setup --verbose
Checking if a JVM is installed
Found a JVM installed under /Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home.

Checking if ~/Library/Application Support/Coursier/bin is in PATH

Checking if the standard Scala applications are installed
Found app ammonite in channel io.get-coursier:apps
➜  ~ 
```

Increasing the verbosity does not seem to help, and I'm having some trouble figuring out the exact issue I'm facing as a result. I'd like to ask if there is a way to get a more verbose error message here, or if not, if messages could be added in the future. Thanks in advance!

**Comment by sloshy** [comment]
Quick follow-up: I've resolved my specific underlying issue by setting some java properties. If it helps identify the area where the logs are lacking, I had to set proxy settings via `-J-Dhttp.proxyHost=...`, etc. So it was unable to connect to the hosts for getting each artifact. After figuring that out, I also had to set the `COURSIER_REPOSITORIES` environment variable properly; I had set it before, but I neglected to remove `central` from there as I'm behind a corporate artifactory instance.

So to summarize my personal issue is solved, but it would be great if the logs were more helpful to find the specific problem, if possible.
